QuadradiusR/scripts/lobby_manager.gd

```diff
@@ -5,13 +5,19 @@ onready var player_list = $PlayerList
 
 
 func _ready():
-	player_list.add_player("Stefan GUEST", true)
-	player_list.add_player(NetworkHandler.username, false)
 	if not NetworkHandler.lobby_data['players'].empty():
-		for i in NetworkHandler.lobby_data['players']:
-			player_list.add_player(i['id'], false)  # TODO request own id too
+		for player in NetworkHandler.lobby_data['players']:
+			player_list.add_player(player['username'], player['id'], false)
+	player_list.add_player(NetworkHandler.username, "", true)  # TODO add uuid properly
```

Comment:
Maybe add a call of method `NetworkHandler.rest_api.get_user_me()` and insert the to player list so we do not forget to do that later?